samples/openapi3/server/petstore/springboot-useoptional/src/main/java/org/openapitools/model/AdditionalPropertiesAnyType.java

```diff
@@ -24,25 +26,24 @@
 public class AdditionalPropertiesAnyType extends HashMap<String, Object>  {
 
   @JsonProperty("name")
-  private String name;
+  private Optional<String> name = Optional.empty();
 
   public AdditionalPropertiesAnyType name(String name) {
-    this.name = name;
+    this.name = Optional.ofNullable(name);
```

Comment:
Should be just Optional.of().

I like the spring boot way all parameter are non null by default and if you would like to use a null parameter you mark it with @Nullable. 

So i would use Optional.of and if you would like to reset the Optional you have to use the setter!